types/googlemaps/index.d.ts

```diff
@@ -1995,6 +1996,10 @@ declare namespace google.maps {
         onAdd(): void;
         onRemove(): void;
         setMap(map: Map | StreetViewPanorama | null): void;
+        /** @see {@link https://developers.google.com/maps/documentation/javascript/reference/overlay-view#OverlayView.preventMapHitsAndGesturesFrom Maps JavaScript API} */
+        static preventMapHitsAndGesturesFrom(element: Element): void;
+        /** @see {@link https://developers.google.com/maps/documentation/javascript/reference/overlay-view#OverlayView.preventMapHitsFrom Maps JavaScript API} */
```

Comment:
```suggestion
        /**
         * Stops click or tap on the element from bubbling up to the map. Use this to prevent the map from triggering "click" events.
         * @see {@link https://developers.google.com/maps/documentation/javascript/reference/overlay-view#OverlayView.preventMapHitsFrom Maps JavaScript API}
         */
```